app/views/content_advice_requests/_request_details.html.erb

```diff
@@ -2,34 +2,78 @@
   <p><%= f.object.class.description %></p>
 </div>
 
-<%= f.input :title, label: "Title of request", input_html: { class: "input-md-6", placeholder: "Advice on content" } %>
+<div class="form-group">
+  <span class="form-label">
+    <%= f.label :title, "Title of request" %>
+  </span>
+  <span class="form-wrapper">
+    <%= f.text_field :title, placeholder: "Advice on content", class: "input-md-6 form-control" %>
+  </span>
+</div>
+
+<fieldset>
+  <legend>
+    <span>Details of your request</span>
+  </legend>
 
-<%= f.inputs name: "Details of your request" do %>
-  <%= f.input :details, as: :text, required: true,
-              label: "Please explain what you would like help with",
-              input_html: { class: "input-md-6", rows: 6, cols: 50, :"aria-required" => true } %>
+  <div class="form-group">
+    <span class="form-label">
+      <%= f.label :details do %>
+        Please explain what you would like help with<abbr title="required">*</abbr>
+      <% end %>
```

Comment:
In the app I was working on they did this kind of thing another way -

```
      <%= f.label :justifications, "Is this need in proposition for GOV.UK? You can tick more than one.<br/><span class='optional'>(optional)</span>".html_safe %>
```

I don't know if one way is preferable